### ISSUE TITLE
Reverse transition:

### ISSUE THREAD

**Issue by f00dagi** [icanos/hassio-plejd#26]
Hi icanos

I think that I have found two additional bugs related to transition:.

The transition to make a light source brighter from lets say 10 to 254 over 20 seconds works fine...
```
entity_id: light.lounge_tak
brightness: 254
transition: 20
```
...but going in the other direction from 254 to 10 over 20 seconds results in a direct jump to 10.
```
entity_id: light.lounge_tak
brightness: 10
transition: 20
```
Also if the light source is already at brightness 100 and should with transition move to 254 it seems to start at lowest brightness before moving to 254 while the expected behavior would be starting at 100 and move to 254.

I have tested this with an IKEA trådfri light source and that works fine.

Thanks again for your efforts!

-Daniel

**Comment by icanos** [comment]
Will look into it more, the implementation that's currently in there is quite rudimentary. I'll add that to my backlog, but won't be fixed until I can iron out the stability issues. Will get back to you :-)

**Comment by jzee** [comment]
I hit a similar, if not related issue: when I add transition values, the transition is executed as commanded but it's extremely jerky - even with units that sit directly next to the pi. I can get about one brightness change a second. 

Is the transition something that needs to be executed by the add-on or can the dimmer itself take a transition value so that the add-on would only forward it and the transition could be smoother?

**Comment by icanos** [comment]
I'm going to revisit this now as it seems that 0.3.x works a lot better.

**Comment by icanos** [comment]
Resolved in #64
